webapp/channels/src/actions/websocket_actions.jsx

```diff
@@ -770,8 +770,13 @@ async function handlePostDeleteEvent(msg) {
         const thread = getThread(state, post.root_id);
         if (thread) {
             const userId = getCurrentUserId(state);
-            const teamId = getCurrentTeamId(state);
-            dispatch(fetchThread(userId, teamId, post.root_id, true));
+            let teamId = getTeamIdFromPost(state, post.id);
```

Comment:
0/5 but it might make sense for `getTeamIdFromPost` to automatically return the current team ID for a DM/GM instead of doing it here. I'm trying to think of if we'd always want that behaviour whenever it's used

Reply:
Yes I was thinking about that, will do